android_release/11.0.yml

```diff
@@ -0,0 +1,126 @@
+release:
+  release_date: '2025-07-14'
+  text: >
+    **These notes apply to Thunderbird for Android version 11.0 released July 14th, 2025.**
+
+  releases:
+    - version: '11.0'
+      release_date: '2025-07-14'
+
+notes:
+  # New
+  - note: Drop-down drawer is enabled by default
+    short_note: Drop-down drawer is enabled by default
+    tag: new
+
+  - note: Improve dark mode emails with algorithmic darkening
+    short_note: Improve dark mode emails
+    tag: new
+
+  - note: Welcome screen logo layout improved
+    short_note: Welcome screen layout improved
+    tag: new
+
+  # Changed
+  - note: Selection now shown on active items by adjusting color priority
+    short_note: Selection now shown by adjusting color priority
+    tag: changed
+
+  - note: Folder settings now use switches instead of checkboxes
+    short_note: Folder settings now use switches
+    tag: changed
+
+  - note: Message list colors updated for better contrast
+    short_note: Message list colors updated
+    tag: changed
+
+  - note: Messages are now marked read by default when archiving
+    short_note: Messages are now marked read by default when archiving
+    tag: changed
+
+  - note: Font size config preference is removed
+    short_note: Font size config preference is removed
+    tag: changed
+
+  # Fixed
+  - note: Account setup issues with outlook.com and hotmail.com resolved
+    short_note: Account setup issues with outlook and hotmail resolved
+    tag: fixed
+
+  - note: Keyboard navigation bugs resolved
+    tag: fixed
+
+  - note: Star and attachment icon colors corrected
+    short_note: Star and attachment icon colors corrected
+    tag: fixed
+
+  - note: Paste of recipient lists into compose field now correctly parsed
+    tag: fixed
+
+  - note: Back button now returns to Drafts folder when editing a draft
+    tag: fixed
+
+  - note: Unified Inbox now appears as expected
+    tag: fixed
+
+  - note: Dark and light mode switching issues resolved
+    tag: fixed
+
+  - note: Scroll indicator removed following review feedback
+    tag: fixed
+
+  - note: System navigation bar did not correctly blend with message content
+    short_note: System navigation bar did not correctly blend with message content
+    tag: fixed
+
+  - note: '"New Mail Notifications" setting did not persist'
+    short_note: '"New Mail Notifications" setting did not persist'
+    tag: fixed
+
+  - note: Trailing whitespace in recipient fields caused "Incomplete input" error
+    short_note: Trailing whitespace in recipient fields caused "Incomplete input" error
+    tag: fixed
+
+  - note: Navigation drawer "Show Unified inbox" setting did not work
+    short_note: Navigation drawer "Show Unified inbox" setting did not work
+    tag: fixed
+
+  - note: Application could crash after deleting accounts and continuing to use the app
```

Comment:
...no longer crashes... sounds a bit nicer than "could crash". I initially read "could crash" as if it were a feature that it could crash :D